### PR TITLE
Darken dropdown divider

### PR DESCRIPTION
**Pull request by patrickhlauke**
currently, `$gray-200` is practically imperceptible. darkening to `$gray-600` which also roughly matches the dropdown border color (when against white)

### CODE REVIEW

**Comment by patrickhlauke** [comment]
Current look of divider:

> divider-gray-200

After this change:

> divider-gray-600

**Comment by patrickhlauke** [comment]
no backport, as it seems the divider wasn't a problem in v4 https://getbootstrap.com/docs/4.5/components/dropdowns/#dividers

**Comment by ysds** [comment]
LGTM, but also has the option of referring to `$dropdown-border-color`.

File: scss/_variables.scss [hunk 1]
```diff
@@ -914,7 +914,7 @@ $dropdown-border-color:             rgba($black, .15) !default;
 $dropdown-border-radius:            $border-radius !default;
 $dropdown-border-width:             $border-width !default;
 $dropdown-inner-border-radius:      subtract($dropdown-border-radius, $dropdown-border-width) !default;
-$dropdown-divider-bg:               $gray-200 !default;
+$dropdown-divider-bg:               $gray-600 !default;
```
Comment (mdo) [inline scss/_variables.scss:917]: Per @ysds's comment, let's roll with the dropdown's border.

```suggestion
$dropdown-divider-bg:               $dropdown-border-color !default;
```

Comment (patrickhlauke) [reply scss/_variables.scss]: Done (separately, as I hadn't even clocked this suggested change commit thing from the email)

**Comment by patrickhlauke** [comment]
Using border color:

> dropdown-divider-using-border-color

**Review by mdo — APPROVED** [review]

**Pull request merged**